### DISCUSSION TITLE
Issues when running Automate tests on Safari version 15-17

### DISCUSSION THREAD

**Discussion by noSpamsPlease** [cypress-io/cypress#28160] [Questions and Help] [answered]
Is anyone facing issues when running Automate tests on safari version 15-17?

I have my setup running for Safari version 14 but with the latests version something has changed most the calls are failing. 

Any insight here would help. 

Thanks.

**Comment by grandEL-dsouza** [comment, marked as answer]
Yes, something in the encoding mechanism seems to have changed. 

I have faced an issue where in most of my calls fail with either error 304 or 302

This usually happens if I run my tests on a remote device. 

There is an observation however, if I disable the NSURL as seen in this screenshot it works for me:
<img width="1136" alt="image" src="https://github.com/cypress-io/cypress/assets/60457320/de8e8fbe-7baa-442e-b51b-7f52f664d426">

So there seems to be something with this feature it does not happen on my internal network only occurs on a remote device. 

This is likely cause some protocol is breaking or some encoding is failing. 

I hope this helps. 

Thanks.

**Reply by noSpamsPlease** [reply]
Would need to find a Cypress equivalent for this.

Thanks for the information.
